concent_api/core/message_handlers.py

```diff
@@ -443,6 +455,10 @@ def handle_send_force_subtask_results(client_message: message.concents.ForceSubt
     provider_public_key = hex_to_bytes_convert(task_to_compute.provider_public_key)
     requestor_public_key = hex_to_bytes_convert(task_to_compute.requestor_public_key)
 
+    validate_golem_message_signed_with_key(
+        client_message.ack_report_computed_task,
+        requestor_public_key,
+    )
```

Comment:
What about `report_computed_task` in this case ?

Reply:
same as above.